.github/workflows/lint.yml

```
@@ -9,6 +9,6 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
       - name: Lint Code Base
-        uses: github/super-linter@v3.14.0
+        uses: github/super-linter@v3.14.4
```

```suggestion
        uses: github/super-linter@v3
```

Should be updated to `v3` as the tag is automatically updated to point on the latest release (such thing as using `actions/checkout@v2`).

---

I've witnessed multiple occurrences of `super-linter` doing breaking changes on minor releases, so I prefer to see things break in a dependabot PR instead of in an unrelated change that would make me loose time 🙂